.ado/pipelines/templates/jobs-workload-deploy.yaml

```diff
@@ -182,7 +179,7 @@ jobs: # using jobs so they each run in parallel
             # Apply workload BackgroundProcessor helm chart
             echo  "*** Deploy workload BackgroundProcessor to $aksClusterName (in $aksClusterResourceGroup) via helm chart"
             helm upgrade --install workload-backgroundprocessor src/app/charts/backgroundprocessor `
-                         --namespace $(workloadNamespace) --create-namespace `
+                         --namespace $(workloadNamespace) --create-namespace --wait `
```

Comment:
--wait comes already from this (although I would actually vote to drop the parameter since we are never using it and just always specify --wait where we want it

parameters:
  additionalHelmArguments:     '--wait'

Reply:
cannot remember why we've introduced `additionalHelmArguments` in the first place. It's kind of useless here anyhow given how different the workloads deployed in `jobs-workload-deploy.yaml` are. It's not really an abstract and reusable template like most of the others.

Reply:
yeah then just remove it